lib/l10n_utils/tests/test_template.py

```diff
@@ -125,3 +127,53 @@ def test_render_no_locale(self, django_render, get_lang_path):
         # Note: no .locale on request
         # Should not cause an exception
         render(request, '500.html')
+
+
+@patch.object(l10n_utils, 'django_render')
```

Comment:
We also have to patch `template_is_active`.

```python
@patch('lib.l10n_utils.template_is_active', Mock(return_value=True))
@patch('lib.l10n_utils.django_render')
```
That set of patches works for me.